microcosm_flask/audit.py

```diff
@@ -45,6 +46,33 @@ def should_skip_logging(func):
     return getattr(func, SKIP_LOGGING, False)
 
 
+@contextmanager
```

Comment:
I'm fine with using that header name. I think boolean value + root logger makes sense.